app/[user]/page.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image';
+// These imports just for grabbing the static data - these should be removed
+import path from 'path';
+import fs from 'fs';
+
+export const metadata = {
+  title: 'Dashboard'
+}
+
+const profileDirectory: string = path.join(process.cwd(), 'data');
+
+// Grab all users, this would probably query the user database
```

Comment:
Use [TSDoc](https://tsdoc.org/) syntax for doc comments on functions

Reply:
Will do!